src/workerd/api/http.h

```diff
@@ -463,12 +464,31 @@ class Fetcher: public jsg::Object {
   jsg::Promise<void> delete_(jsg::Lock& js, kj::String url,
       CompatibilityFlags::Reader featureFlags);
 
+  struct ServiceBindingQueueMessage {
+    // Representation of a queue message for use when invoking the queue() event handler on another
+    // worker via a service binding.
+    kj::String id;
+    kj::Date timestamp;
+    jsg::Value body;
```

Comment:
I'd prefer to avoid any mechanism that accepts `ArrayBuffer` in one environment but not in the production environment, and I'd prefer not to be passing the serialized form around directly.

Reply:
> There aren't too many structured cloneable types, although some like ReadableStreams and WebAssembly.Modules might be tricky. Does Queues support these?

No, just what's documented here: https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm

I've tested to confirm that `ReadableStream`s do not work in a `queue.send(request.body)` call, and I got `Error: #<ReadableStream> could not be cloned.` (as desired)